### ISSUE TITLE
Use read command instead of regular variable assignment

### ISSUE THREAD

**Issue by kaushalmodi** [kaushalmodi/eless#17]
This would allow the use of back-quotes and single quotes (and everything in general) without needing to use escapes for constructing the whole emacs config blob.

Reference: http://crowding.github.io/blog/2014/08/16/replace-less-with-emacs/

**Comment by kaushalmodi** [comment]
Briefly experimented with using `read`, but it did not work as the elisp code being read in contains a `` ` ``, thus giving this error:

> bad substitution: no closing "`" in `[ *]" (buffer-name buf)) ;Do not count buffers with names starting with space or *

It is this line:

```
(unless (string-match "\\`[ *]" (buffer-name buf)) ;Do not count buffers with names starting with space or *
```

**Comment by kaushalmodi** [comment]
An alternative is to use [this trick][1] by @alphapapa as suggested [here][2].

[1]: https://github.com/alphapapa/matrix-client.el/blob/1e118e903a106c164c8610687160c33a90e309c8/matrix-client-standalone.el.sh#L63-L65
[2]: https://www.reddit.com/r/emacs/comments/cjr9hi/emacscalc_or_ec_is_a_script_that_lets_you_run_the/evh9qk1/

**Comment by alphapapa** [comment]
Found it on StackOverflow.  Works really well!  :)

**Comment by kaushalmodi** [comment]
@alphapapa Looks like you need to escape just the `` ` ``, right?

**Comment by alphapapa** [comment]
I'm not completely sure what you're referring to, but if you mean what I think you do, then yes, I think that's right.  :)

**Comment by kaushalmodi** [comment]
I was referring to:

https://github.com/alphapapa/matrix-client.el/blob/1e118e903a106c164c8610687160c33a90e309c8/matrix-client-standalone.el.sh#L76

**Comment by alphapapa** [comment]
Right, the backquote is a metacharacter, even inside quoted strings and heredocs.  I forget about that, because I always use `$()`.  That bit me today when working on https://github.com/alphapapa/makem.sh, inside an elisp docstring.

If I were to rewrite the code at that URL, I would probably use `(list...` instead of backquoting and unquoting in that list.
